mlir/include/mlir/ExecutionEngine/CRunnerUtils.h

```diff
@@ -486,6 +486,10 @@ extern "C" MLIR_CRUNNERUTILS_EXPORT void *rtsrand(uint64_t s);
 extern "C" MLIR_CRUNNERUTILS_EXPORT uint64_t rtrand(void *, uint64_t m);
 // Deletes the random number generator.
 extern "C" MLIR_CRUNNERUTILS_EXPORT void rtdrand(void *);
+// Returns a pointer to an array of random numbers in the range of [0, s).
```

Comment:
please document that it needs the generator as parameter (looking at L486, using "g" as name and documenting it would have been helpful, but that is of course not your change)

Reply:
Also, the in the range [0,s) is confusing. The numbers themselves are in the range [0,m) per the generator constructor. But this method simply returns an array of length s. Perhaps make that more clear too.

Reply:
Sounds good! I modified line #486 as well. Hopefully that's okay.